app/views/criteria/_criterion.html.erb

```diff
@@ -20,13 +20,13 @@
   </span>
   <span class='right' id='visibility_indicator_<%= criterion.class %><%= criterion.id %>'>
     <% if criterion.ta_visible && criterion.peer_visible %>
-      <i class="fa-solid fa-chalkboard-user icon-left"></i>
-      <i class="fa-solid fa-comments"></i>
+      <i class="fa-solid fa-chalkboard-user icon-left" title="Graders and Instructors"></i>
```

Comment:
In general if you want to add new strings, you should do so in an appropriate locale file (https://guides.rubyonrails.org/i18n.html).

But in this case you should reuse the existing strings and not create new strings.